src/main/java/edu/byu/cs/controller/ConfigController.java

```diff
@@ -44,6 +44,29 @@ public class ConfigController {
         return "";
     };
 
+    public static final Route scheduleShutdown = (req, res) -> {
+        User user = req.session().attribute("user");
+
+        Gson gson = new Gson();
+        JsonObject jsonObject = gson.fromJson(req.body(), JsonObject.class);
```

Comment:
What do you mean make this a proper class?

In #480 I'll be looking at making all config info in one (or two) Config Data Transfer Objects. This may or may not solve the problem you're talking about.

I didn't know that Serializer was a thing. I'll go change that.